lib/iser.c

```diff
@@ -711,6 +711,14 @@ iscsi_iser_send_pdu(struct iscsi_context *iscsi, struct iscsi_pdu *pdu) {
 	iscsi_pdu_set_expstatsn(pdu, iscsi->statsn + 1);
 	ISCSI_LIST_ADD_END(&iscsi->waitpdu, pdu);
 
+	/* because of async reconnection, before reconnecting successfully,
+	   the argument 'iscsi' is the 'old_iscsi' with a empty connection.
+	 */
+	if (!iser_conn) {
+		ISCSI_LOG(iscsi, 2, "iscsi_iser_send_pdu: queuing pdu to old_iscsi while reconnecting");
```

Comment:
Please change this message into something that is comprehensible for users who are not familiar with the libiscsi source code.

Reply:
There is already a log msg in iscsi_scsi_command_async, how about removing log here?